docs/en/Configuring-Alluxio-with-S3.md

```diff
@@ -91,6 +91,16 @@ The `jets3t` version `0.9.0` works for Hadoop version `2.3.0`. The `jets3t` vers
 work for older versions of Hadoop. To find the exact `jets3t` version for your Hadoop version,
 please refer to [MvnRepository](http://mvnrepository.com/).
 
+## Use non-amazon service provider
+
+If you have a service provider other than "s3.amazonaws.com", modify the `ALLUXIO_JAVA_OPTS` section of `conf/alluxio-env.sh` to include:
+
+{% include Configuring-Alluxio-with-S3/non-amazon.md %}
+
+For these parameters, replace `<S3_ENDPOINT>` with the host name of your S3 service. Set `<USE_HTTPS>` to `true` or `false`, then set the appropriate port number; without those values the system will default to 80 for HTTP and 443 for HTTPS. You can remove the unused port entry. 
```

Comment:
A few comments:
- Is it the case that users only need to set `alluxio.underfs.s3.endpoint` property if they don't want to use `s3.amazonaws.com`, right? If this is the case, please be more explicit here. (also a side question: does it work if we set `alluxio.underfs.s3.endpoint` to be `s3.amazonaws.com`?)
- "without those values the system will default to 80 for HTTP and 443 for HTTPS", please be more explicit here what are "those values", as we have talked a few properties. It is always better in documentation to be clear and explicit, even sometimes it is a bit verbose :)
- "You can remove the unused port entry." what do we mean here?